### ISSUE TITLE
[Vanilla Bug] Deploy hotkey doesn't command aircraft transports to unload units.

### ISSUE THREAD

**Issue by MikillRosen** [Vinifera-Developers/Vinifera#103]
### Description:
Deploy hotkey doesn't work on aircraft transports, only the mouse does.

### Expected Behaviour:
The deploy hotkey commands an aircraft transport to unload units.

### Actual Behaviour:
Only the mouse can command an aircraft transport to unload units

**Comment by Crimsonum** [comment]
> Perhaps this could be a QoL improvement and enabled as standard?

Agreed. Given that there are no buildable air transports in multiplayer, this won't change the existing meta.

**Comment by CCHyper** [comment]
Should the Aircraft also try to land if the Deploy command is issued and it is currently in the air?
